x-pack/plugins/monitoring/server/plugin.ts

```diff
@@ -79,6 +80,7 @@ export class Plugin {
   }
 
   async setup(core: CoreSetup, plugins: PluginsSetup) {
+    AlertingSecurity.init(plugins.encryptedSavedObjects);
```

Comment:
You can add this to the route dependencies here: https://github.com/elastic/kibana/pull/71846/files#diff-eef915a6b8dba3b904a62609b073163aR205